.gitignore

```diff
@@ -12,3 +12,4 @@
 ###> lexik/jwt-authentication-bundle ###
 /config/jwt/*.pem
 ###< lexik/jwt-authentication-bundle ###
+.php-cs-fixer.cache
```

Comment:
Manque une ligne à la fin

Reply:
Une explication de pourquoi cette norme: https://unix.stackexchange.com/a/18789/67970

Tu peux configurer PHPStorm pour qu’il ajoute une ligne vide automatiquement à chaque fin de fichier:
File | Settings | Editor --> Ensure blank line before end of file on Save
:)